docs/editor/accessibility.md

```diff
@@ -70,179 +70,181 @@ In `settings.json` file, nest the following code inside the outermost curly brac
+|   Tritanopia     |   Reduced sensitivity to blue light. This condition is considered rare.
+|   Monochromia    |   The inability to see all colors, also referred to as *achromatopsia*. More information about the rarest form of colorblindness: [Foundation Fighting Blindness](https://www.fightingblindness.org/diseases/achromatopsia).
 
 One of the best approaches to selecting the best colors for a specific condition is to apply complementary colors. These are colors located opposite of one another on a color wheel.
 
-![A color wheel highlighting complementary colors for regular vision, deuteranopia, protanopia, tritanopia and monochromacy](images/accessibility/accessibility-color-wheels.png)
+![Color wheel highlighting complementary colors for regular vision, deuteranopia, protanopia, tritanopia and monochromacy](images/accessibility/accessibility-color-wheels.png)
 
->**Note**: For more information on finding complementary colors, go to [Adobe Color](https://color.adobe.com/create/color-accessibility) to access the color blind simulator and interactive color wheel.
+>**Note**: For more information on finding complementary colors, access the colorblind simulator and interactive color wheel at [Adobe Color](https://color.adobe.com/create/color-accessibility).
 
 ## Dim unfocused editors and terminals
 
-Unfocused views can be dimmed to make it clearer where typed input will go. This is especially useful when working with multiple editor groups or terminals. Enable this feature by setting `"accessibility.dimUnfocused.enabled": true`. You can control the dimness level with `accessibility.dimUnfocused.opacity`, which takes the opacity fraction from 0.2 to 1 (default 0.75).
+Unfocused views can be dimmed to make it more clear where typed input will go. This is especially useful when working with multiple editor groups or terminals. Turn this feature on by setting `"accessibility.dimUnfocused.enabled": true` and configure the dimness level with `accessibility.dimUnfocused.opacity`.
 
 ## Keyboard navigation
 
-You will find that VS Code provides an exhaustive list of commands in the **Command Palette** (`kb(workbench.action.showCommands)`) so that you can run VS Code without using the mouse.  Press `kb(workbench.action.showCommands)` then type a command name (for example 'git') to filter the list of commands.
+VS Code includes an exhaustive list of commands in the **Command Palette** (`kb(workbench.action.showCommands)`) so that you can use VS Code without a mouse.  Select `kb(workbench.action.showCommands)`, then type a command name (for example, "git") to filter the list of commands.
 
 VS Code also has many preset keyboard shortcuts for commands.
 
-![Keybindings for commands are displayed at the end of the command palette entry](images/accessibility/keyboard-shortcuts.png)
+![Key bindings for commands are displayed at the end of the command palette entry](images/accessibility/keyboard-shortcuts.png)
```

Comment:
While perhaps not fully correct, we've landed on keybindings in the VS Code documentation